roles/kubernetes-apps/csi_driver/azuredisk/templates/azure-csi-azuredisk-controller.yml.j2

```diff
@@ -24,6 +24,10 @@ spec:
           operator: "Equal"
           value: "true"
```

Comment:
remove `value: true` and `operator: Equal`

on modern cluster value is `""`, no `"true"`

Reply:
done